app/presenters/hyrax/menu_presenter.rb

```diff
@@ -0,0 +1,32 @@
+module Hyrax
+  # view-model for the admin menu
+  class MenuPresenter
+    def initialize(view_context)
+      @view_context = view_context
+    end
+
+    attr_reader :view_context
+
+    delegate :controller, :controller_name, :action_name, :content_tag,
+             :current_page?, :link_to, to: :view_context
+
+    # Returns true if the current controller happens to be one of the controllers that deals
+    # with workflow.  This is used to keep the parent section on the sidebar open.
+    def workflows_section?
+      controller.instance_of? Hyrax::Admin::WorkflowRolesController
```

Comment:
No, I don't think so. One would override this whole method, as this condition could get much more complex (checking action_name, multiple controllers, etc.)